scripts/copy-assets.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+set -o errexit
+
+mkdir -p -- dist/{fonts,images} dist/fonts/72
+ 
+for file in scss/icons/*.{eot,svg,ttf,woff}; do cp "$file" dist; done
```

Comment:
Would there ever be other file extensions we'd want to move?  Or maybe better asked ... are there certain extensions we are omitting such that we can't just copy all the file from one directory to the other?